pkg/utils/httper/httper.go

```diff
@@ -95,7 +95,7 @@ func Post(url string, data []byte, contentType string, head map[string]string) (
 		req.Header.Add(k, v)
 	}
 	if err != nil {
-		panic(err)
+		return ""
```

Comment:
How is this "proper error handling"

Error should be bubbled up, so that whomever did the `Post` can handle it.

Empty content might be ok in some cases